cf-oscore/src/test/java/org/eclipse/californium/oscore/ContextRederivationTest.java

```diff
@@ -124,6 +130,26 @@ public void rederivationTest() throws OSException, ConnectorException, IOExcepti
 		
 		assertEquals(resp.getCode(), ResponseCode.CONTENT);
```

Comment:
The output of junit test are much easier to understand, if the parameters are provided in the expected order documented in the test-method. here: expected, actual.
if you provide actual, expected, that results in wired messages as 

> java.lang.AssertionError: expected:<4.04> but was:<2.05>

Reply:
Fixed in commit cab5b8639dc31aa6ec31b0747a631d127b50c993